### PR TITLE
fix: increase check_delay to address exception in arbiter callback

### PR DESCRIPTION
**Pull request by aarnphm**
Screenshot 2023-02-08 at 01 08 13

Signed-off-by: Aaron Pham <29749331+aarnphm@users.noreply.github.com>

### CODE REVIEW

File: src/bentoml/_internal/utils/circus/__init__.py [hunk 1]
```diff
@@ -53,5 +53,9 @@ def create_standalone_arbiter(watchers: list[Watcher], **kwargs: t.Any) -> Arbit
                 watchers,
                 endpoint=f"tcp://127.0.0.1:{endpoint_port}",
                 pubsub_endpoint=f"tcp://127.0.0.1:{pubsub_port}",
+                # NOTE: increase check delay to 5s to avoid
+                # ConfclitError raised when arbiter is running arbiter_stop command
+                # See https://github.com/bentoml/BentoML/pull/3540.
+                check_delay=5,
```
Comment (sauyon) [inline src/bentoml/_internal/utils/circus/__init__.py:59]: I think this could use some more explanation, either here or in the PR description.

**Comment by parano** [comment]
How do you reproduce this error?

**Comment by aarnphm** [comment]
> How do you reproduce this errors?

After a long serving period, when exiting this should show up.

**Comment by aarnphm** [comment]
I will probably take another look to fix this, instead of having this patch fix. Closing for now

**Pull request closed without merging**